packages/common/src/intl/IntlTestProvider.tsx

```diff
@@ -4,11 +4,13 @@ import { LocaleMessages, SupportedLocales } from './intlHelpers';
 import en from './locales/en.json';
 import fr from './locales/fr.json';
 import pt from './locales/pt.json';
+import ab from './locales/ab.json';
 
 const locales: Record<SupportedLocales, LocaleMessages> = {
   en,
   fr,
   pt,
+  ab,
```

Comment:
`ab` is the code for arabic and arabic is rtl, right?

Reply:
I thought it was `ar`
Closes to a definitive list I can find is the [unicode cldr](https://github.com/unicode-org/cldr-json/tree/master/cldr-json/cldr-localenames-full)